tests/kafkatest/services/connect.py

```diff
@@ -156,7 +164,7 @@ def node(self):
     def start_cmd(self, node, connector_configs):
         cmd = "( export KAFKA_LOG4J_OPTS=\"-Dlog4j.configuration=file:%s\"; " % self.LOG4J_CONFIG_FILE
         cmd += "export KAFKA_OPTS=%s; " % self.security_config.kafka_opts
-        cmd += "/opt/%s/bin/connect-standalone.sh %s " % (kafka_dir(node), self.CONFIG_FILE)
+        cmd += "%s %s " % (self.path.script("connect-standalone.sh", get_version(node)), self.CONFIG_FILE)
```

Comment:
This is where I think mixins might actually be handy -- all the services could just have these methods mixed in and then they could grab the version automatically so we wouldn't need to litter the code with `get_version` calls.